php/commands/core.php

```diff
@@ -134,7 +134,8 @@ public function download( $args, $assoc_args ) {
 		$locale = \WP_CLI\Utils\get_flag_value( $assoc_args, 'locale', 'en_US' );
 
 		if ( isset( $assoc_args['version'] ) ) {
-			$version = $assoc_args['version'];
+			$version = strtolower( $assoc_args['version'] );
+			$version = ( 'nightly' === $version ? 'trunk' : $version );
 			$download_url = $this->get_download_url($version, $locale, 'tar.gz');
```

Comment:
When `version=trunk`, we're passing `tar.gz` as a file type and getting `zip` in return. Will that cause bugs later on?